src/libstd/prelude/v1.rs

```diff
@@ -105,8 +105,8 @@ pub use core::prelude::v1::{
     test_case,
 };
 
-// The file so far is equivalent to src/libcore/prelude/v1.rs,
-// and below to src/liballoc/prelude.rs.
+// The file so far is equivalent to `src/libcore/prelude/v1.rs`,
+// and below to `src/liballoc/prelude/v1.rs` (except `HashMap`).
 // Those files are duplicated rather than using glob imports
 // because we want docs to show these re-exports as pointing to within `std`.
```

Comment:
Should I remove these comments or not? @RalfJung

Reply:
Sorry, no idea. I just observed an inconsistency.